src/zoid/card-fields/prerender.jsx

```diff
@@ -5,61 +5,67 @@ import { node, type ChildType } from "@krakenjs/jsx-pragmatic/src";
 
 type PrerenderedCardProps = {|
   nonce: ?string,
-  height: ?number,
+  height: ?number | ?string,
+  disableDefaultStyle?: boolean,
 |};
 
-const DEFAULT_HEIGHT = 78;
+export const DEFAULT_HEIGHT = 78;
 
 export function CardPrerender({
   nonce,
   height,
+  disableDefaultStyle,
 }: PrerenderedCardProps): ChildType {
   return (
     <html>
       <body>
         <style
           nonce={nonce}
-          innerHTML={`
-                        html, body {
-                            padding: 0;
-                            margin: 0;
-                            width: 100%;
-                            height: 100%;
-                        }
+          innerHTML={
+            disableDefaultStyle
```

Comment:
We decided to not display the prerender template when the `disableDefaultStyle` flag is passed since we now rely on the container style. That style is what will be displayed while the card renders. 

If we did not do this, the existing pre-render style may not be a good experience.